### PR TITLE
qc-24: Fix js a12 rule

### PR DESCRIPTION
**Pull request by liza-veis**
https://github.com/BinaryStudioAcademy/quality-criteria/issues/24

### CODE REVIEW

**Review by what1s1ove — APPROVED** [review]
Please, pull the latest changes to your branch

File: src/javascript.md [hunk 1]
```diff
@@ -1280,7 +1280,7 @@
 
 - <details>
     <summary>
-      <b>A12.</b> Where possible, the ternary operator is used in the assignment of a value instead of if.
+      <b>A12.</b> Where possible, the ternary operator is used in the assignment of a value instead of if. But ternary operator should not be nested.
```
Comment (what1s1ove) [inline src/javascript.md]: it should be in details

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by what1s1ove** [comment]
Please, pull the latest changes to your branch

File: src/javascript.md (hunk 1 above)
Comment (liza-veis) [reply src/javascript.md]: done

**Review by what1s1ove — APPROVED** [review]
LGTM!

**Pull request merged**